assets/chat/js/autocomplete.js

```diff
@@ -197,6 +197,22 @@ class ChatAutoComplete {
         this.selected = -1
         this.results = []
         this.criteria = criteria
+
+        //for emote suffixes started from "YEE :"
+        if (criteria.orig.includes(' :') && criteria.word.startsWith(':')) {
+            const lastColon = criteria.orig.lastIndexOf(' :')
+            //assumption: the last occurrence of " :" is meant to be a emote suffix
+            criteria.orig = criteria.orig.substring(0, lastColon) + ':' + criteria.orig.substring(lastColon + 2)
```

Comment:
Is this for removing the space before the colon character?

Reply:
yes